samples/rust/rust_chrdev.rs

```diff
@@ -16,8 +16,7 @@ module! {
     author: b"Rust for Linux Contributors",
     description: b"Rust character device sample",
     license: b"GPL v2",
-    params: {
-    },
+    alias: b"samples_rust_chrdev",
```

Comment:
Instead of adding an alias to this sample, I would improve the docs of the `module!` macro to include an explanation/example of all the optional keys etc.

Reply:
Added it to the documentation, could you please check for spelling mistakes :-)